src/assets/style/module/changelog/_changelog.less

```diff
@@ -0,0 +1,97 @@
+.changelog-container {
+    width: 480px;
+    max-height: 480px;
+    overflow-y: scroll;
+    margin-left: -240px;
+    padding: 12px 15px;
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    background-color: @dark;
+    color: @light-gray;
+    font-size: 12px;
+    font-family: inherit;
+    opacity: 0;
+    border-radius: 1px;
+    box-sizing: border-box;
+    box-shadow: 0 0 0 6px @blue-medium;
+    pointer-events: none;
+    transition: opacity 0.5s ease;
+    z-index: 200;
+}
+
+.changelog-container.changelog-container-is-visible {
+    pointer-events: auto;
+    opacity: 1;
+}
+
+.changelog-container .changelog-title {
+    min-height: 1.5em;
+    padding: @spacing-2x 0 7px 0;
+    font-size: 170%;
+    text-align: center;
+    box-sizing: border-box;
+}
+
+.changelog-container .changelog-version {
+    min-height: 1.5em;
+    padding: @spacing-1x 0 4px 0;
+    font-size: 140%;
+    text-align: center;
+    box-sizing: border-box;
+}
+
+.changelog-container .changelog a {
+    color: @light-gray;
+}
+
+.changelog-container .dismiss-changelog {
+    min-height: 1.5em;
+    padding: @spacing-1x 0 4px 0;
+    font-size: 140%;
+    text-align: center;
+    text-decoration: underline;
+    color: @light-gray;
+    box-sizing: border-box;
+}
+
+/*
+    The headers are automatically built from markdown files,
+    so we can't select them by classes
+*/
+.changelog h1,
+.changelog h2,
+.changelog h3,
+.changelog h4,
+.changelog h5,
+.changelog h6 {
+    font-size: 16px;
+    padding-top: 15px;
+}
+
+.changelog li {
+    padding: 15px 0 0 0;
+}
+
+.changelog li + li {
+    padding: 5px 0;
+}
+
+.changelog a {
+    color: @light-gray;
+}
+
+.changelog table,
```

Comment:
do we have tables in the changelog?  or is this just copied from the airportGuide stuff?

Reply:
IMHO we should use the same styling for the airport guides and the changelog.

Reply:
agreed.  maybe move that to a different css module and call it `blackBox` or something?